src/styles/blocks/_catalog.scss

```diff
@@ -0,0 +1,23 @@
+.catalog {
+  display: grid;
+  justify-items: center;
+  grid-template-columns: repeat(auto-fill, minmax(200px, 1fr));
```

Comment:
I don't think that we need to use minmax here. Your cards are too far apart, the gap between them should be fixed at 48 px. It is better to use @media and fixed columns amount+size